schema/component.schema.json

```diff
@@ -0,0 +1,210 @@
+{
+  "$anchor": "youtube-component",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "$merge": {
+    "source": {
+      "$ref": "component"
+    },
+    "with": {
+      "required": [
+        "_media"
+      ],
+      "properties": {
+        "_supportedLayout": {
```

Comment:
I think `_supportedLayout` object is missing the enum values for `half`, `full`, `both` as per the [media](https://github.com/adaptlearning/adapt-contrib-media/blob/master/schema/component.schema.json#L15) plugin